Source/Flow/Public/Interfaces/FlowCoreExecutableInterface.h

```diff
@@ -0,0 +1,63 @@
+// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#pragma once
+
+#include "UObject/Interface.h"
+
+#include "FlowCoreExecutableInterface.generated.h"
+
+// Implemented by objects that can execute within a flow graph
+// (eg, UFlowNode and UFlowNodeAddOn subclasses implement this)
+UINTERFACE(MinimalAPI, Blueprintable, DisplayName = "Flow Core Executable Interface")
+class UFlowCoreExecutableInterface : public UInterface
+{
+	GENERATED_BODY()
+};
+
+class FLOW_API IFlowCoreExecutableInterface
```

Comment:
This is the interface for the "Core" Flow node & add-on functions.  

These are core to the experience of an executable element of a flow graph.

Reply:
I have doubts about whether we need this separate interface. It's already included in `UFlowNodeBase` so all node types are using it. 

I might not see a reason for extracting this code to the interface. Is it perhaps useful in internal projects?
If there's no important reason for that, it would be worth moving this code back to the node class, as this would simplify understanding code for smaller teams and ease future maintenance.

Reply:
We need "IFlowCoreExecutableInterface" and "IFlowExternalExecutableInterface" for the differences between internal and external flow executable cases.

As for the "IFlowNativeExecutableInterface", I scanned the code and couldn't find any compelling reason why it must be an interface as opposed to simply virtual functions on UFlowNodeBase.

I don't think we need to keep IFlowNativeExecutableInterface.

I can undo this, but I'd like to merge from Flow-mainline first to avoid a nasty merge.  And we'd have to change our code to suit, since we would need to replace references of IFlowNativeExecutableInterface with UFlowNodeBase references.